scripts/ensure-stale-workflow-is-in-all-repos.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+root="$(realpath "$(dirname "$0")/..")"
+
+repository_file="$(jq '.' "$root/github/ipfs/repository_file.json")"
```

Comment:
q: do we need this `repository_file.json` at all? 
we have repo list in `repository.json` already, and path is the same everywhere (`.github/workflows/stale.yml`), can we remove `repository_file.json` ?

Reply:
The existence of both `repository.json` and `repository_file.json` comes from the GitHub Management design. Each of those files relates to a specific Terraform GitHub Provider resource it describes and it closely resembles how Terraform itself represents resources in Terraform state (thanks to this it's easy for us to sync back and forth between this repository and actual GitHub configuration). 

`repository.json` describes [`github_repository` resources](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository), `repository_file.json` describes [`github_repository_file` resources](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository_file). You can also have a look at the [full list of resources supported by GitHub Management](https://github.com/ipfs/github-mgmt/blob/master/docs/ABOUT.md#supported-resources) currently.

All in all, they're separate JSONs because they describe different GitHub resources - repositories and files respectively. We could try to combine the two - for example by introducing a `files` key inside repository objects in `repository.json` - but this would require a bit of work in the `scripts/sync.sh`, `terraform/resources.tf` and `docs/*` so I would rather not tackle it as part of this PR. Something we could revisit in the future though for sure.